.storybook/main.js

```diff
@@ -119,6 +119,9 @@ const config = {
                 require("postcss-preset-env")({
                   stage: 1,
                   preserve: true,
+                  features: {
```

Comment:
Storybook was checking for ["functional notation" of rgb and hsl values](https://cssdb.org/#color-functional-notation), which is interesting because the hsl values in this PR are not written in that format.

Reply:
What it was checking for:

`hsl(100deg 50% 50%)`
`hsla(100deg 50% 50% / 0.75)`

How we're writing the values:

`hsl(100deg, 50%, 50%)`
`hsla(100deg, 50%, 50%, 0.75)`